numba_dppy/dpnp_glue/__init__.py

```diff
@@ -4,3 +4,6 @@ def ensure_dpnp(name):
         from . import dpnp_fptr_interface as dpnp_glue
     except ImportError:
         raise ImportError("dpNP is needed to call np.%s" % name)
+
+
+DEBUG = None
```

Comment:
Is it used?

Reply:
Duplication of DEBUG configuration

DEBUG variable is in files:
- numba_dppy/dpnp_glue/__init__.py
- numba_dppy/compiler.py
- numba/core/config.py

It should be in files:
- numba_dppy/config.py
- numba/core/config.py

numba-dppy DEBUG should be set as numba DEBUG by default.
It should be able to set via NUMBA_DPPY_DEBUG env variable.

Reply:
Moved to ticket https://github.com/IntelPython/numba-dppy/issues/211